perf/benchmark/run_benchmark_job.sh

```diff
@@ -254,7 +254,6 @@ for dir in "${CONFIG_DIR}"/*; do
     # TRIALRUN as a pre-submit check, only run agaist the first set of enabled perf run in the perf_conf file
     if [[ "${TRIALRUN}" == "True" ]]; then
        run_benchmark_test "${WD}/configs/trialrun.yaml"
-       collect_flame_graph
```

Comment:
why delete this? we could also have this tested in the presubmit job I do think

Reply:
so to make the above perf_record enabled?

Reply:
Let's focus on the main part(installation, setup, runner) first

Reply:
ok, other cases can be added in the following PRs.